lib/json_logger.rb

```diff
@@ -0,0 +1,14 @@
+require 'logger'
+require 'json'
+
+# Logs to stdout. All the time.
```

Comment:
Might be worth having a TODO so that anything `:warn` level and above goes to `stderr` maybe...? Definitely low priority, and may not be the right thing to do.